pennylane/numpy/random.py

```diff
@@ -26,8 +26,14 @@
 
 wrap_arrays(_random.__dict__, globals())
 
+# np_version = semantic_version.version("numpy")
+
+# # Ensure pre/post release tags are compatible with SemVer
+# if any((match := s) in np_version for s in ["rc", "dev", "post"]):
+#     np_version = (f"-{match}").join(np_version.split("rc"))

```

Comment:
Is this meant to be removed?

Reply:
This is something I was still waiting to get @mlxd comment on since I got it from his original PR and I am not sure if it's needed.

Reply:
This can be removed --- it was used temporarily as the numpy wheels were not semver compliant. We can add this in without problem, but safe to assume wheels should comply.

Reply:
Sounds good, I will erase it then.